pydm/widgets/baseplot.py

```diff
@@ -676,7 +688,6 @@ def to_dict(self) -> OrderedDict:
                 ("label", self._label),
                 ("minRange", self.range[0]),
                 ("maxRange", self.range[1]),
-                ("autoRange", self._auto_range),
```

Comment:
This line should be restored (using `self.auto_range` instead I think looks fine). This method is used in building the `.ui` file that gets saved, so having `autoRange` here is a property is needed so that setting it in Designer still works. Otherwise it will always load as false.